src/align.cpp

```diff
@@ -21,13 +21,13 @@
 static chunk_t *align_var_def_brace(chunk_t *pc, size_t span, size_t *nl_count);
 static chunk_t *align_trailing_comments(chunk_t *start);
 static void align_init_brace(chunk_t *start);
-static void align_func_params();
+static void align_func_params(void);
```

Comment:
some c++ compilers still complain about the old C syntax when set to their highest warning level and void shuts them up